Makefile

```diff
@@ -52,13 +52,13 @@ po/$(PACKAGE_NAME).js.pot:
 		--keyword=gettext:1,1t --keyword=gettext:1c,2,2t \
 		--keyword=ngettext:1,2,3t --keyword=ngettext:1c,2,3,4t \
 		--keyword=gettextCatalog.getString:1,3c --keyword=gettextCatalog.getPlural:2,3,4c \
-		--from-code=UTF-8 $$(find src/ \( -name '*.js' -o -name '*.jsx' \) \! -path 'src/lib/*')
+		--from-code=UTF-8 $$(find src/ -name '*.js' -o -name '*.jsx')
 
 po/$(PACKAGE_NAME).html.pot: $(NODE_MODULES_TEST) $(LIB_TEST)
-	src/lib/html2po.js -o $@ $$(find src -name '*.html' \! -path 'src/lib/*')
+	pkg/lib/html2po.js -o $@ $$(find src -name '*.html')
 
 po/$(PACKAGE_NAME).manifest.pot: $(NODE_MODULES_TEST) $(LIB_TEST)
-	src/lib/manifest2po.js src/manifest.json -o $@
+	pkg/lib/manifest2po.js pkg/manifest.json -o $@
```

Comment:
For follow up, this is now ported to Python, so take note when you bump COCKPIT_REPO_COMMIT, you'd likely want to do that as separate step